go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/anchore/grype
 
-go 1.21
+go 1.21.0
 
-toolchain go1.21.0
+toolchain go1.21.1
```

Comment:
I think @willmurphyscode latest PR adjusts the `go` minimum version to 1.21.1 and removes the `toolchain` keyword entirely https://github.com/anchore/grype/pull/1484